modules/light-clients/08-wasm/keeper/genesis_test.go

```diff
@@ -99,17 +70,18 @@ func (suite *KeeperTestSuite) TestExportGenesis() {
 	suite.SetupTest()
 	ctx := suite.chainA.GetContext()
 
+	expCodeID := "c64f75091a6195b036f472cd8c9f19a56780b9eac3c3de7ced0ec2e29e985b64"
```

Comment:
I guess there is no way to avoid the magic string?

Reply:
One way would be to make an exported wrapper for `generateWasmCodeHash` only for testing. I can look into this in a separate PR.